newsfragments/928.misc.rst

```diff
@@ -0,0 +1 @@
+On win32 we no longer set SO_EXCLUSIVEADDRUSE when binding a socket in :exc:`trio.open_tcp_listeners`. See https://github.com/python-trio/trio/issues/39, https://github.com/python-trio/trio/issues/72, and https://github.com/python-trio/trio/issues/928 for the gory details.
```

Comment:
I think you can drop the second sentence. When we generate the release notes, the `928` in the filename will be automatically converted into a link to #928 (you can [see this in action in the release notes](https://trio.readthedocs.io/en/latest/history.html); it's configured by [this bit of magic in our pyproject.toml](https://github.com/python-trio/trio/blob/00de6cb8fdc7cf96d17d015369be11feb3fc4abe/pyproject.toml#L12)). And then if anyone follows the link to #928, they'll see links to #39 and #72 right in the first message.

Reply:
Thanks, got it.